README.md

```
@@ -13,6 +13,21 @@ Stocky is an [MCP](https://modelcontextprotocol.io) server that lets an AI
 assistant search, inspect and download royalty-free stock photography from
 **Pexels** and **Unsplash**.
 
+> ### 🆕 Version 2.0 — already using Stocky? Read this
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)